stl/inc/mdspan

```diff
@@ -705,6 +703,17 @@ public:
     using data_handle_type = typename accessor_type::data_handle_type;
     using reference        = typename accessor_type::reference;
 
+    static_assert(
+        sizeof(_ElementType) > 0, "ElementType must be a complete type (N4928 [mdspan.mdspan.overview]/2.1).");
+    static_assert(
+        !is_abstract_v<_ElementType>, "ElementType cannot be an abstract type (N4928 [mdspan.mdspan.overview]/2.1).");
+    static_assert(
+        !is_array_v<_ElementType>, "ElementType cannot be an array type (N4928 [mdspan.mdspan.overview]/2.1).");
+    static_assert("Extents must be a specialization of extents (N4928 [mdspan.mdspan.overview]/2.2).");
+    static_assert(is_same_v<_ElementType, typename _AccessorPolicy::element_type>,
+        "Expression is_same_v<ElementType, typename AccessorPolicy::element_type> should be true (N4928 "

```

Comment:
I'm not sure if there's already an established convention, but I'd prefer "must", or similar imperative wording, instead of "should". Occurs below.